src/ert/storage/local_ensemble.py

```diff
@@ -247,32 +246,8 @@ def load_response(self, key: str, realizations: Tuple[int, ...]) -> xr.Dataset:
         assert isinstance(response, xr.Dataset)
         return response
 
-    def load_field(self, key: str, realizations: List[int]) -> npt.NDArray[np.double]:
-        result: Optional[npt.NDArray[np.double]] = None
-        for index, realization in enumerate(realizations):
-            input_path = self.mount_point / f"realization-{realization}"
-            if not input_path.exists():
-                raise KeyError(
-                    f"Unable to load FIELD for key: {key}, realization: {realization}"
-                )
-            data = np.load(input_path / f"{key}.npy", mmap_mode="r")
-            data = data[np.isfinite(data)]
-
-            if result is None:
-                result = np.empty((len(realizations), data.size), dtype=np.double)
-            elif data.size != result.shape[1]:
-                raise ValueError(f"Data size mismatch for realization {realization}")
-
-            result[index] = data
-
-        if result is None:
-            raise ConfigValidationError(
-                "No realizations found when trying to load field"
-            )
-        return result.T
-
     def field_has_data(self, key: str, realization: int) -> bool:
-        path = self.mount_point / f"realization-{realization}/{key}.npy"
+        path = self.mount_point / f"realization-{realization}/{key}"
```

Comment:
Might add some info to still have an extension.

Reply:
https://github.com/equinor/ert/issues/5650

I have created an issue to use the same extension for all parameter-types.
The reason I don't want to do it in this PR is because it effects GEN_KW and Surfaces in addition to FIELDs.